src/components/style/index.css

```diff
@@ -11,6 +11,9 @@
 /* Variables */
 @import "./vars/buttonColors.css";
 
+/* Reach framework styles */
+@import "@reach/menu-button/styles.css";
```

Comment:
How much of what they define here are we actually using?

Reply:
I considered that, but we want to stay in sync with the library as it gets updated.

Part of the api design for Reach is to include base styles for each component, so Reach should be the point of truth for those base styles instead of us defining a separate one. Our file is for overrides only.